Sources/MessagingPush/PushHandling/PushEventHandlerProxy.swift

```diff
@@ -57,16 +57,26 @@ class PushEventHandlerProxyImpl: PushEventHandlerProxy {
             // Each iteration of the loop waits for the push event to be processed by the delegate.
             for delegate in nestedDelegates.values {
                 await withCheckedContinuation { continuation in
+                    // Some SDKs, like the rn-firebase SDK (RNFBMessagingUNUserNotificationCenter), will call the completionHandler twice.
+                    // Handle that by only responding to the first call and ignoring the rest.
+                    var hasResumed = false
```

Comment:
See [the integration tests](https://github.com/customerio/customerio-ios/pull/735/files#r1633746365) to understand why this is needed.